File: 2006Scape Server/plugins/plugin/click/item/ItemThirdClick.java
package plugin.click.item;

import com.rs2.event.EventContext;
import com.rs2.event.EventSubscriber;
import com.rs2.event.SubscribesTo;
import com.rs2.event.impl.ItemThirdClickEvent;
import com.rs2.game.players.Player;

import static com.rs2.game.content.StaticItemList.YOYO;

@SubscribesTo(ItemThirdClickEvent.class)
public final class ItemThirdClick implements EventSubscriber<ItemThirdClickEvent> {

    @Override
    public void subscribe(EventContext context, Player player, ItemThirdClickEvent event) {
        if (player.playerRights == 3) {
            player.getPacketSender().sendMessage("[ItemClick#3] - ItemId: " + event.getId());
        }

        switch (event.getId()) {

            case YOYO:
                player.startAnimation(1460);
                break;


        }
    }

}

File: 2006Scape Server/plugins/plugin/click/item/ItemThirdClick.kt
package plugin.click.item

import com.rs2.event.EventContext
import com.rs2.event.EventSubscriber
import com.rs2.event.SubscribesTo
import com.rs2.event.impl.ItemThirdClickEvent
import com.rs2.game.content.StaticItemList.YOYO
import com.rs2.game.players.Player

@SubscribesTo(ItemThirdClickEvent::class)
class ItemThirdClick : EventSubscriber<ItemThirdClickEvent> {

	override fun subscribe(context: EventContext, player: Player, event: ItemThirdClickEvent) {

		if (player.playerRights >= 3) {
			player.packetSender.sendMessage("[ItemClick#3] - ItemId: ${event.id}")
        }
		
		when(event.id) {

			YOYO -> player.startAnimation(1460)


		}
		
		
	}
	
}